### PR TITLE
fix: addTextComponentScaleform is not a function

### PR DESCRIPTION
**Pull request by Germancitoz**
And added posiblity to add native sound.

### CODE REVIEW

**Comment by C0kkie** [comment]
The 2nd parameter of endTextCommandDisplayHelp is also a boolean

File: docs/articles/snippets/helptext.md [hunk 1]
```diff
@@ -9,10 +9,10 @@ You cannot change the position of this text.
...
 alt.onServer('showHelpText', showHelpText);
 
-export function showHelpText(text, milliseconds) {
-    native.beginTextCommandDisplayHelp('STRING');
-    native.addTextComponentScaleform(text);
-    native.endTextCommandDisplayHelp(0, 0, 0, milliseconds);
+export function showHelpText(text, sound, milliseconds) {
+  native.beginTextCommandDisplayHelp("STRING");
+  native.addTextComponentSubstringPlayerName(text);
+  native.endTextCommandDisplayHelp(0, 0, sound, milliseconds);
```
Comment (C0kkie) [inline docs/articles/snippets/helptext.md:15]: 2nd parameter is also a boolean

Comment (Germancitoz) [reply docs/articles/snippets/helptext.md]: fixed

**Comment by C0kkie** [comment]
Thank you

**Pull request merged**